.kokoro/build.bat

```diff
@@ -1,8 +1,39 @@
 REM This file runs tests for merges, PRs, and nightlies.
 REM There are a few rules for what tests are run:
-REM  * PRs run all non-acceptance tests for every library.
-REM  * Merges run all non-acceptance tests for every library, and acceptance tests for all altered libraries.
-REM  * Nightlies run all acceptance tests for every library.
-REM Currently only runs tests on 2.5.1
+REM  * PRs run all non-acceptance tests for every library, against all rubies.
+REM  * Merges run all non-acceptance tests for every library, and acceptance tests for all altered libraries, against all rubies.
+REM  * Nightlies run all acceptance tests for every library, against all rubies.
 
-"C:\Program Files\Git\bin\bash.exe" github/google-cloud-ruby/.kokoro/windows.sh
+SET /A ERROR_CODE=1
+
+CD C:\
+
+REM Rubocop expects native line endings
+REM TODO: set this in Dockerfile or change .rubocop.yml or skip rubocop on windows
+git config --system --unset core.autocrlf
+git config --global core.autocrlf true
+
+REM Ruby can't access the files in the mounted volume.
+REM Neither Powershell's Copy-Item nor xcopy correctly copy the symlinks.
+REM So we clone/checkout the repo ourselves rather than relying Kokoro.
+SET "run_kokoro=bundle update && bundle exec rake kokoro:%JOB_TYPE%"
```

Comment:
Should this command also call `ridk enable`?

Reply:
yeah. It was hanging on that earlier. playing around with it now